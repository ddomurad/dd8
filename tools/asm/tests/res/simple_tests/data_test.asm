.org 0x1000
.db 0x18, 0x23, "SOME TEST"
.dw 0x99, 0x111, 0x1234, "12345"
.db "111", "222"
.dw "1", "22", "333"
.db "new\nline"
