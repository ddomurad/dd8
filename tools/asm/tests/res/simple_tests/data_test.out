:201000001823534F4D45205445535499001101341231323334350031313132323231003285
:0D10200032333333006E65770A6C696E65FC
:00000001FF
